crates/pop-cli/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "pop-cli"
+description = "An all-in-one tool for Polkadot development."
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "pop"
+path = "src/main.rs"
+
+[dependencies]
+anyhow.workspace = true
+duct.workspace = true
+tempfile.workspace = true
+url.workspace = true
+tokio.workspace = true
+
+# pop-cli
+clap.workspace = true
+cliclack.workspace = true
+console.workspace = true
+strum.workspace = true
+strum_macros.workspace = true
+
+# contracts
+pop-contracts = { version = "0.1.0", path = "../pop-contracts", optional = true}
```

Comment:
For local crates `version` may be omitted.

Reply:
True!